packages/orchestration/src/utils/tx.js

```diff
@@ -0,0 +1,66 @@
+// @ts-check
+import { TxBody } from '@agoric/cosmic-proto/cosmos/tx/v1beta1/tx.js';
+import { decodeBase64, encodeBase64 } from '@endo/base64';
+
+/** @typedef {string} Base64Bytes - Uint8Array, base64 encoded */
```

Comment:
i'm ambivalent about type aliases for primitive types. I don't think this one is carrying its weight. 

I could see `Proto3Msg` being made generic with the typeUrl, so that tools could convey that type you would get out upon decoding. but I see that as polish later, lower priority than getting the features working.

consider removing this type and relying on JSDoc instead to explain the contents

Reply:
I personally see value in this alias, we have `Bytes` from `@agoric/network` to capture the pattern where the first 8 bits are string encoded. Seems valuable to have something more than a comment capture "base64 encoded uint8array"

Removed for now, but might lobby to re-add it in the future.